From 622e70a7953f9801f3c41ab240df0c547f074dcd Mon Sep 17 00:00:00 2001
From: Daniel Cederman <cederman@gaisler.com>
Date: Fri, 4 Sep 2020 09:55:56 +0200
Subject: [PATCH 08/50] common: Create builtin define __BIG_ENDIAN__

---
 gcc/config/sparc/sparc-c.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/gcc/config/sparc/sparc-c.c b/gcc/config/sparc/sparc-c.c
index 108e8fa876e..925af2e2e3a 100644
--- a/gcc/config/sparc/sparc-c.c
+++ b/gcc/config/sparc/sparc-c.c
@@ -31,6 +31,8 @@ sparc_target_macros (void)
 {
   builtin_define_std ("sparc");
 
+  cpp_define (parse_in, "__BIG_ENDIAN__");
+
   if (TARGET_ARCH64)
     {
       cpp_assert (parse_in, "cpu=sparc64");
-- 
2.34.1

